packages/kbn-content-management/table_list/src/table_list_view/actions.ts

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
nit: we could consider a types package, but I'm not necessarily requiring that.

This file is all types, though, you might consider `types/actions.ts`.

Reply:
Yes. Although I do like to have the `action.ts` sit next to the `reducer.tsx` file as they are directly related. Will leave it as it for now but might reconsider later